### ISSUE TITLE
Dropdown losing initial value after focus.

### ISSUE THREAD

**Issue by DieterDS1983** [telerik/kendo-angular#1104]
## I'm submitting a...
* Bug report 

## Current behavior
When the application loads initial values in a dropdown component, it is correctly displayed after load.
When the dropdown gets the focus and loses it again, the value of the dropdown is set to null.
This happens only after initial load. If the user selects an item in the dropdown, losing focus will not set it to null.

## Expected behavior
The value of the dropdown should remain unchanged after losing focus.

## Minimal reproduction of the problem with instructions
See Plunker: http://plnkr.co/edit/EXoWfoUHKp2MMdsj6Te8?p=preview
There are 2 ways to reproduce:

1. After initial load, use "Tab" to toggle between controls.
If a dropdown loses focus, the value is changed to null.

2. After initial load, select a dropdown to open the list of values.
Then click outside the dropdown without selecting an item.

## Environment

Package versions:
TypeScript: 2.6.1
Angular: 5.0.2
kendo-angular-dropdowns: 1.2.4

Browser:
- Chrome (desktop) version XX

System:
- TypeScript version: 2.6.1
- Node version: 7.5.0
- NPM version: 4.1.2
- Platform:  Windows

**Comment by tapopov** [comment]
Hi @DieterDS1983 

I managed to reproduce this and can confirm it's not expected. We are currently investigating what's causing it.

**Comment by sboyd** [comment]
Hello @tapopov I was about to create a similar bug. The same thing happens inside of the filter menu.

If you go to https://www.telerik.com/kendo-angular-ui-develop/components/grid/data-operations/filtering/

Scroll down to the Filter Menu section
Click on the "Unit Price" filter
Change "Is equal to" to any other value
Click inside the text box 
You'll notice the same behavior.

Let me know if you'd like me to open another ticket.
--Steve

**Comment by tapopov** [comment]
@DieterDS1983 
This should be fixed in `v1.2.5-dev.201711221500`

@sboyd 
Thanks for reporting this. We're working on it.

**Comment by sboyd** [comment]
Awesome @tapopov! Any idea when it'll be ready for me to pull into my app?

**Comment by DieterDS1983** [comment]
@tapopov 
Thanks. Any idea when the final v1.2.5 will be released?

**Comment by tapopov** [comment]
Both issues should be fixed in the official v1.2.5 expected by the end of the week.

**Comment by sboyd** [comment]
Awesome!

I'm assuming that the dev site https://www.telerik.com/kendo-angular-ui-develop/components/grid/data-operations/filtering/ hasn't been updated yet with the latest version of the dropdown.

**Comment by tapopov** [comment]
We've released official `kendo-angular-dropdowns` v1.3.0. The latest package version should fix the issues described above. The kendo-angular-ui-develop site however is not yet updated.

**Comment by sboyd** [comment]
@tapopov FANNNNNTASTIC I just installed the latest version and it looks great.
Thank you! --Steve

**Comment by DieterDS1983** [comment]
I can confirm. 
Installed this version this morning and everything is working fine.

Thanks.

**Comment by tsvetomir** [comment]
The latest version of the `@progress/kendo-angular-dropdowns` package is 3.0.3 - not sure where 3.10 came from.

Can you please try updating to it to see if that helps? Remove package-lock.json before installing the new version as it may be holding on to an older version.

**Comment by Attosius** [comment]
On 3.0.3  error occurs again, but way to reproduce changed:

1. Open dropdown
2. Write something in the filter to find value, that not in "view"
3. Select item
4. Click somewhere else
5. valueChange fired, value = defaultItem
**This reproduces only if [defaultItem] is set.**

**Comment by tapopov** [comment]
Hi @Attosius 
I will reopen the issue until we verify it. Stay tuned.

**Comment by dimitar-pechev** [comment]
@Attosius Thank you for your example.

The described issue is fixed in the latest [dev build](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/) (**3.4.1-dev.201811231129**) of the `@progress/kendo-angular-dropdowns` package.

**Comment by dimitar-pechev** [comment]
The fix is included in **v3.4.1** of the `@progress/kendo-angular-dropdowns` package.
